### ISSUE TITLE
reportAttributeAccessIssue accessing Enum values in 1.1.398

### ISSUE THREAD

**Issue by palfrey** [microsoft/pyright#10226]
**Describe the bug**

In 1.1.397, the following is fine:
```python

class Units(Enum):
    DAYS = 'DAYS'
    HOURS = 'HOURS'

Units.HOURS
```

In 1.1.398 we get
```
error: Cannot access attribute "HOURS" for class "type[Units]"
    Attribute "HOURS" is unknown (reportAttributeAccessIssue)
```

**VS Code extension or command-line**

This is all from the `pyright` commandline tool.

**Comment by palfrey** [comment]
*sigh*. False alarm. The units code was actually now
```
class Units(Enum):
    days = 'DAYS'
    hours = 'HOURS'
```
because it's generated code and the generator broke. Apologies.
